src/TypeSpec.Extension/Emitter.Csharp/src/lib/utils.ts

```diff
@@ -1,11 +1,20 @@
-import { Model } from "@typespec/compiler";
+import { Model, getFriendlyName, getProjectedNames } from "@typespec/compiler";
+import {
```

Comment:
Do we have the code style check enabled to sort the imports?

Reply:
Yes. there is prettier format to control this and it is a check in the pipeline.